main.rb

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+#
+# Bubble sort and Bubble sort by
+#
+def bubble_sort(array)
+  for i in (0...array.length - 1)
+    for i in (0...array.length - 1)
+      array[i], array[i + 1] = array[i + 1], array[i] if array[i] < array[i + 1]
```

Comment:
The default sort provides a sorter order in increasing order. But the method you have generated provides decreasing order. Please convert it as returns increasing order sorted array.